testplan/testing/multitest/entries/base.py

```diff
@@ -285,5 +285,7 @@ def __init__(self, graph_type, graph_data, description=None,
             self.discrete_chart = True
         elif graph_type in valid_graph_types:
             self.discrete_chart = False
+        else:
+            self.type = 'UnimplementedGraph - ' + graph_type
```

Comment:
So what happens here when we try to render such an UnimplementedGraph in the UI or in the PDF exporter?

Reply:
In the UI:
The default for an 'type' that is not yet implemented will be to display a message saying that this assertion has not yet been implemented and to contact the developers if you want it to be.
(This gets handled in Assertion.js, and this logic gets applied to ever type not implemented, so i just needed to make it an type not recognized in Assertion.js)

In the PDF:
Here nothing except the description gets displayed, as by the default again of an un-recognized/unimplemented type in base.py in exporters

Reply:
I think I would prefer to raise a ValueError here instead of just accepting it and not rendering. I'm thinking that it's quite likely a user could mistype the graph name ("Scattre" etc.) and then be confused why their graph does not get displayed. Raising an exception straight away makes it much more obvious.